### ISSUE TITLE
[suggestion/complaint] Music Discs

### ISSUE THREAD

**Issue by SimulatorSia** [TheCyberBrick/The-Betweenlands#524]
I have cleared out at least 6 crag towers and 2 fortresses.
Spent hours upon hours in boats and walking though biomes in search of chests and pots. I've gotten over a hundred item scrolls and at least a couple loot hundred chests and this loot is always the same stuff I get. I haven't even gotten 1 music disc. Am I doing something wrong? I notice there are 13 of them added but I've gotten none.
This is tedious and aggravating.

Also the explorer's hat. I thought for sure while clearing out two fortresses I'd at least find one but none.

Just these
https://imgur.com/a/ESG3z

**Comment by PetasiGit** [comment]
If you look at the [loot tables](http://the-betweenlands.wikia.com/wiki/Loot_Tables), they're only obtainable from shrines, towers, fortresses, and item scrolls. They're also one of the rarest items, having the same rarity as a ring of ascent or a ring of power.

I've cleared out at least 9 of each and still don't have them all. They're rare, yes, but you'll get them eventually. 

edit: Also, using boats is horribly inefficient. If you're not in an old version, try using Harlequin Toads to get around.

**Comment by TheCyberBrick** [comment]
Rebalancing the loot tables is on our TODO list :+1:

**Comment by SimulatorSia** [comment]
Maybe those items being this rare is a good idea. It feels so much more rewarding when you do find one. Or you find a hat and ring of ascent in the same shrine chest o.o

**Comment by alxcool** [comment]
... I was about to create a topic on the same matter. I have a map mod, so I could count 9 completed cragrock tower + 1 wight fortress so far, in a 3.3.10 / 3.3.11 map. I've earn the following result. 1 Disc in a fortress chest. 0 In cragrock towers... Most of the time tower's chest (when available) lend only low grade drops... like common chests loot. The best loots in a tower came from the pot loots! I've animated enough Item scroll to recover 3 disc, (probably 75.)
3 time in a row it was Frosty Disc.?!? Looted probably 16 Shrines. Let say shrines loots usually are quite good, but doesn't contain any disc, so far. So I don't want to complain but this looks like a bug.

As suggestion to fix this, maybe create a recipe or system who allow to trade rare items for items scrolls? And about disc from item scroll, 3 Frosty disc in a row, 1 week between each of them WTF??? This may require to be tested?

Edit : Nice I din't know we could complete 2 wight fortress on the same single player game ^^.
Here! to help with this topic, I've passed some stats test about item scroll drops... Here are a summary followed by more data : Note % are for comparison with %weight
http://the-betweenlands.wikia.com/wiki/Item_Scroll

Voodoo Doll = 29/200 (14,4%)
Hag Hacker = 20/200  (10.25%)
Critter Crusher = 22/200 (10.25%)
Sludge Slicer = 19/200 (10.25%)
Amulet Slot = 25/200 (10.25%)
Swift Pick = 13/200 (10.25%)
Valonite Shard = 26/200 (10.25%)
Wight Bane = 19/200 (8.2%)
Skull Mask = 14/200 (8.2%)
Angry Pebble = 7/200 (4%)
Ring of Power = 4/200 (2.6%)
Music Disc = 2/200 (1%) Beneat a Green Sky/Temple Disc Ancient

Passed twice 100 item scroll in 10 different animators...
Item Scroll 200
A)
Voodoo Doll 7
 Hag Hacker / Critter Crusher 4/2
 Sludge Slicer / Amulet Slot 1/3
 Valonite Shard 1 (4)
 Wight Bane 1
 Angry Pebble 1 (29)
B)
Voodoo Doll 2
 Hag Hacker / Critter Crusher 3/5
 Sludge Slicer / Amulet Slot 2/2
 Swift Pick 1
 Wight Bane / Skull Mask 2/2
 Angry Pebble 1 (4)
C)
 Hag Hacker / Critter Crusher 2/2
 Sludge Slicer / Amulet Slot 1/2
 Valonite Shard 3 (8/3/5)
 Swift Pick 3
 Wight Bane / Skull Mask 2/3
 Angry Pebble 2 (27/4)
D)
Voodoo Doll 1
 Hag Hacker /  3
 Sludge Slicer / Amulet Slot 1/3
 Valonite Shard 5 (3/5/3/6/3)
 Swift Pick 3
 Wight Bane / Skull Mask 2/1
 Angry Pebble 1 (18)
E)
Voodoo Doll 4
 Critter Crusher 2
 Sludge Slicer / Amulet Slot 4/5
 Valonite Shard 2 (5/7)
 Wight Bane / Skull Mask 2/1
F)
Voodoo Doll 4
 Hag Hacker / Critter Crusher 0/3
 Sludge Slicer / Amulet Slot 3/1
 Valonite Shard 4 (8/7/6/5)
 Swift Pick 1
 Wight Bane / Skull Mask 1/1
 Ring of Power 1
* Music disc (Temple Disc Ancient)
G)
Voodoo Doll 1
 Hag Hacker / Critter Crusher 1/3
 Sludge Slicer / Amulet Slot 1/5
 Valonite Shard 1 (7)
 Swift Pick 1
 Wight Bane / Skull Mask 3/3
* Musci Disc (Beneat a Green Sky)
H)
Voodoo Doll 3
 Hag Hacker / Critter Crusher 3/1
 Sludge Slicer / Amulet Slot 1/2
 Valonite Shard 4 (4/5/5/6)
 Wight Bane / Skull Mask 2/2
 Angry Pebble 1 (23)
 Ring of Power 1
I)
Voodoo Doll 4
 Hag Hacker / Critter Crusher 2/2
 Sludge Slicer / Amulet Slot 2/2
 Valonite Shard 1 (5)
 Swift Pick 4
 Wight Bane / Skull Mask 1/1
 Ring of Power 1

J)
Voodoo Doll 3
 Hag Hacker / Critter Crusher 2/2
 Sludge Slicer / Amulet Slot 3/0
 Valonite Shard 5 (5/5/3/3/6)
 Wight Bane / Skull Mask 3/0
 Angry Pebble 1 (18)
 Ring of Power 1
